docs/databases/hadoop/how-to-install-and-set-up-hadoop-cluster/index.md

```diff
@@ -24,21 +24,23 @@ external_resources:
 
 ## What is Hadoop?
 
-Hadoop is an open-source Apache project that allows creation of parallel processing applications on large data sets, distributed across networked nodes. It's composed of the **Hadoop Distributed File System (HDFS™)** that handles scalability and redundancy of data across nodes, and **Hadoop YARN**: a framework for job scheduling that executes data processing tasks on all nodes.
+Hadoop is an open-source Apache project that allows creation of parallel processing applications on large data sets, distributed across networked nodes. It is composed of the **Hadoop Distributed File System (HDFS™)** that handles scalability and redundancy of data across nodes, and **Hadoop YARN**, a framework for job scheduling that executes data processing tasks on all nodes.
```

Comment:
The use of the ":" is strange here, and it's used more as a comma. Avoiding contractions